eZ/Publish/Core/Search/Solr/Content/ResultExtractor/LoadingResultExtractor.php

```diff
@@ -18,6 +18,8 @@
 /**
  * The Loading Extractor extracts the value object from the Elasticsearch search hit data
  * by loading it from the database.
+ *
+ * @todo workshop: implement native extractor (extracting from returned fields)
```

Comment:
Implemented in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/6d13e78ce4fdc7c740f03852cc6821395e393bc7